docs/eventing/samples/container-source/README.md

```diff
@@ -0,0 +1,133 @@
+ContainerSource will start a container image which will generate events under contain situations and send messages to a sink uri. ContainerSource could be an easy way to implement your own event sources. This document shows how to configure ContainerSource as event source for functions and gives instructions on how to create an event source using ContainerSource.
+
+## Create a heartbeats ContainerSource
+
+### Prerequisites
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing and Sources](../../../eventing).
+
+### Prepare the heartbeats image
+
+Knative [event-sources](https://github.com/knative/eventing-sources) has a sample of heartbeats event source. You could clone the source codes by
+```
+git clone https://github.com/knative/eventing-sources.git
+```
+And then build a heartbeats image and publish to your image repo with
+```
+ko publish github.com/knative/eventing-sources/cmd/heartbeats
```

Comment:
In docs I think we should default to using a released image. That way we don't force users to have to install `ko`, clone a repo etc...

Reply:
heartbeats is not an official publish image. After heartbeats images is published officially, it can be removed. But now, I have to include this command here. See the issue https://github.com/knative/eventing-sources/issues/331.